css/system-fonts.css

```diff
@@ -15,13 +15,24 @@
                'franklin gothic medium', 'century gothic', /* Windows stuff */
                'Liberation Sans', /* Linux */
                sans-serif; /* Everything else */ }
+.system-mono { font-family: Menlo, Monaco, /* MacOS */
+              'Segoe UI Mono, Lucida Console', /* Windows */
```

Comment:
Segoe UI Mono didn't come with any version of Windows; it was a part of some XNA addon for Visual Studio.